test/integration/src/test/java/io/pravega/test/integration/WatermarkingTest.java

```diff
@@ -537,7 +478,8 @@ private void scale(Controller controller, Stream streamObj, StreamConfiguration
             currentTime.set(timer.incrementAndGet());
             return writer.writeEvent(count.toString(), currentTime.get())
                     .thenAccept(v -> {
-                        if (count.incrementAndGet() % 10 == 0) {
+                        if (count.incrementAndGet() % 3 == 0) {
```

Comment:
noting time more frequently doesnt change the frequency at which watermarks are emitted on controller.

Reply:
this change ensures noteTime is invoked after 3 seconds instead of the 10 seconds in the previous code.